src/content/plugins/dll-plugin.md

```diff
@@ -23,17 +23,21 @@ This plugin is used in a separate webpack config exclusively to create a dll-onl
 - `context` (optional): context of requests in the manifest file (defaults to the webpack context.)
 - `name`: name of the exposed dll function ([TemplatePaths](https://github.com/webpack/webpack/blob/master/lib/TemplatedPathPlugin.js): `[hash]` & `[name]` )
 - `path`: __absolute path__ to the manifest json file (output)
-- `entryOnly`: if `true`, only entry points will be exposed
+- `entryOnly`: if `true`, only entry points will be exposed (defaults to `true`)
 - `type`: type of the dll bundle
 
 ```javascript
 new webpack.DllPlugin(options);
 ```
 
+Using the DllPlugin without `entryOnly: true` will disable Tree Shaking in the DLL as all exports might be used.
```

Comment:
```suggestion
W> We recommend using DllPlugin only with `entryOnly: true`, otherwise tree shaking in the DLL won't work as all the exports might be used.
```